dracut/modules.d/99kiwi-lib/kiwi-net-lib.sh

```diff
@@ -73,3 +82,8 @@ function _is_compressed {
     local source_url=$1
     [[ ${source_url} =~ .xz$ ]]
 }
+
+function _is_dolly {
+  local source_url=$1
```

Comment:
and four space indentation please :) Thanks

Reply:
I know, I am greedy even with whitespaces...
Fixed.

Reply:
:) thanks